about.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "New Topic button",
-  "about_url": "https://meta.discourse.org/t/new-topic-button-on-all-pages-theme-component/84551",
+  "name": "'New Topic' Header Button",
```

Comment:
Hmm, this will likely mess with alphanumeric sorting

Reply:
I did think it may be a little too avant grade. I'll keep it simple 🙂